src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.*;
 public class Duke {
+    private Scanner sc;
+
+    public Duke(Scanner sc) {
+        this.sc = sc;
+    }
+    public void start(Tasklist list) throws DukeException {
+        while(sc.hasNext()) {
+            String input = sc.nextLine();
+            input = input.trim();
+            String[] separateInput = input.split(" ");
+            String str = separateInput[0];
+            if(input.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                sc.close();
+                return;
+            } else if(input.equals("list")) {
```

Comment:
```suggestion
            } else if (input.equals("list")) {
```
'if' should be followed by a whitespace, do remember to add a whitespace for all the other 'if' and 'else if' too.